torchrl/envs/transforms/transforms.py

```diff
@@ -10437,6 +10442,8 @@ class MultiAction(Transform):
 
     """
 
+    disable_inv_on_reset = True
```

Comment:
PR https://github.com/pytorch/rl/pull/2787 also broke the `ActionDiscretizer`, we should also add the `disable_inv_on_reset = True` into https://github.com/pytorch/rl/blob/c1d894fb6d105c50f99a0d1194a2c339f187e026/torchrl/envs/transforms/transforms.py#L9665

Reply:
I've changed this PR to set `enable_inv_on_reset = False` by default